src/zhinst/toolkit/driver/devices/pqsc.py

```diff
@@ -240,19 +250,28 @@ def find_zsync_worker_port(self, device: BaseInstrument) -> int:
         .. versionadded:: 0.5.1
```

Comment:
this changes the interface of the function so we should add a 

.. versionchanged:: 0.6.1